pagopa/io-app
# feat: [IOBP-840] Add payment tracking property `organization_fiscal_code`

## Short description
This PR adds an additional property `organization_fiscal_code` for the entire payment flow mixpanel events.

## List of changes proposed in this pull request
- Added `organization_fiscal_code` property inside the required screens and events to track the organization fiscal code with the organization name property.

## How to test
- Executing the payment flow, you should be able to see mixpanel events with the organization fiscal code property with a value set.
